src/axolotl/utils/config/models/input/v0_4_1/__init__.py

```diff
@@ -606,6 +606,7 @@ class Config:
         default=None, metadata={"help": {"streaming dataset to use for pretraining"}}
     )
     dataset_processes: Optional[int] = Field(default=os.cpu_count())
+    exact_deduplication: Optional[bool] = None
```

Comment:
Hm, thoughts on labelling this, `dataset_exact_deduplication`?

Reply:
I agree, that would be clearer